src/main/kotlin/com/approvers/devlazaApi/controller/ExtendsFunctions.kt

```diff
@@ -0,0 +1,85 @@
+package com.approvers.devlazaApi.controller
+
+import com.approvers.devlazaApi.errors.BadRequest
+import com.approvers.devlazaApi.errors.NotFound
+import com.approvers.devlazaApi.model.ProjectMember
+import com.approvers.devlazaApi.model.Tags
+import com.approvers.devlazaApi.model.TagsToProjectsBridge
+import com.approvers.devlazaApi.model.Token
+import com.approvers.devlazaApi.repository.ProjectMemberRepository
+import com.approvers.devlazaApi.repository.TagsRepository
+import com.approvers.devlazaApi.repository.TagsToProjectsBridgeRepository
+import com.approvers.devlazaApi.repository.TokenRepository
+import java.util.*
+
+fun String.convertToUUID():UUID{
+    val id: UUID
+    try{
+        id = UUID.fromString(this)
+    }catch (e: IllegalArgumentException){
+        throw BadRequest("The format of id is invalid")
+    }
+    return id
+}
+
+fun String?.divideToTags(): List<String>{
+    val tags: MutableList<String> = this.getTags() ?: return listOf()
+
+    while (tags.indexOf("") != -1){
+        tags.removeAt(tags.indexOf(""))
+    }
+
+    return tags.toList()
+}
+
+fun String?.getTags(): MutableList<String>?{
+    if (this !is String) return null
+
+    val regex = Regex("\\+")
+    val tags: MutableList<String>
+    tags = if (regex.containsMatchIn(this)){
+        this.split("+").toMutableList()
+    }else{
+        this.split(" ").toMutableList()
+    }
+    return tags
+}
+
+fun TokenRepository.getUserIdFromToken(token: String): UUID{
+    val checkedToken: Token = this.tokenCheck(token)
+    return checkedToken.userId
+}
+
+fun TokenRepository.tokenCheck(token: String): Token{
```

Comment:
メソッド名は動詞で書きますね
tokenCheckだとcheckは名詞です
```suggestion
fun TokenRepository.checkToken(token: String): Token{
```